dali/operators/random/rng_base_cpu.h

```diff
@@ -133,6 +134,22 @@ void RNGBase<Backend, Impl, IsNoiseGen>::RunImplTyped(Workspace &ws, CPUBackend)
 
   // TODO(janton): set layout explicitly from the user for RNG
 
+  // Check if _random_state is provided for custom seeding (Dynamic Mode)
+  if (spec_.HasTensorArgument("_random_state")) {
+    const auto &random_state_input = ws.ArgumentInput("_random_state");
+    auto random_state_view = view<const uint32_t>(random_state_input);
+    
+    // Seed each sample's RNG with the provided random state
+    for (int s = 0; s < nsamples; s++) {
+      auto state = random_state_view[s];
+      // Expected: 6 uint32 values per sample
+      DALI_ENFORCE(state.num_elements() >= 6, 
+                   make_string("_random_state must have at least 6 uint32 values per sample, got ",
+                               state.num_elements()));
+      // TODO (janton): use the provided random state
+    }
+  }
```

Comment:
Please, oh please don't touch this even with a one mile stick. I've torn this file apart completely. This will only cause headache with rebasing.